.github/codespell-ignore.txt

```diff
@@ -0,0 +1,4 @@
+crate
+ratatui
+lient
+ody
```

Comment:
Can't find this one in the code base

Reply:
Patterns like this in readme

```
**B**ody
```

And this in code

```
{B}ody
(B)ody
```

You can install codespell locally and launch it, after removing the lines.